src/Fixtures/en.js

```diff
@@ -731,64 +766,6 @@ const breakDrill = {
   ],
 };
 
-const threeColumnsLongDrill = {
```

Comment:
Tou need to apply the same changes to the French version so that it stays up-to-date

Espacially if you're visiting someone from the french federation next week, he'll probably have his phone in french

Reply:
Yes I Know, I didn't have the time yet. Could we merge the PR and I will take the time tomorrow to work on the french version